### ISSUE TITLE
Uncaught Error: ENOENT: no such file or directory, unlink '/var/folders/9h/1j84r06d58z298jf_t96x8...

### ISSUE THREAD

**Issue by fseminario** [armin-pfaeffle/atom-minify#10]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.0.7
**System**: Mac OS X 10.10.4
**Thrown From**: [atom-minify](https://github.com/armin-pfaeffle/atom-minify) package, v0.2.2
### Stack Trace

Uncaught Error: ENOENT: no such file or directory, unlink '/var/folders/9h/1j84r06d58z298jf_t96x8nm0000gp/T/atom-minify.dfbb8d7c-37df-4240-89e7-e0331b85089c.output.tmp'

```
At fs.js:939

Error: ENOENT: no such file or directory, unlink '/var/folders/9h/1j84r06d58z298jf_t96x8nm0000gp/T/atom-minify.dfbb8d7c-37df-4240-89e7-e0331b85089c.output.tmp'
  at Error (native)
  at Object.fs.unlinkSync (fs.js:939:18)
  at AtomMinifier.module.exports.AtomMinifier.deleteTemporaryFiles (/Users/fjs/.atom/packages/atom-minify/lib/minifier.coffee:158:20)
  at /Users/fjs/.atom/packages/atom-minify/lib/minifier.coffee:43:26
  at /Users/fjs/.atom/packages/atom-minify/lib/minifier/css/yui-css.coffee:20:17
  at /Users/fjs/.atom/packages/atom-minify/lib/minifier/BaseMinifier.coffee:12:13
  at ChildProcess.exithandler (child_process.js:736:5)
  at emitTwo (events.js:87:13)
  at ChildProcess.emit (events.js:169:7)
  at maybeClose (child_process.js:997:16)
  at Socket.<anonymous> (child_process.js:1166:11)
  at emitOne (events.js:77:13)
  at Socket.emit (events.js:166:7)
  at Pipe.close (net.js:464:12)

```
### Commands

```
     -0:07.3.0 atom-beautify:beautify-editor (atom-text-editor.editor.is-focused)
     -0:02.7.0 atom-minify:minify-direct (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "disabledPackages": [
      "emmet"
    ],
    "destroyEmptyPanes": false
  },
  "atom-minify": {}
}
```
### Installed Packages

``` coffee
# User
Remote-FTP, v0.7.1
atom-beautify, v0.28.7
atom-minify, v0.2.2
autoclose-html, v0.18.0
color-picker, v2.0.7
filesize, v0.2.0
highlight-line, v0.11.0
language-liquid, v0.2.0
linter, v1.3.4
linter-csslint, v1.0.5
linter-htmlhint, v0.1.1
linter-jshint, v1.1.5
minimap, v4.12.3
wordpress, v0.2.0

# Dev
No dev packages
```

**Comment by armin-pfaeffle** [comment]
Did this error occur more than once? Because before deleting file, code is checking that it exists. So normally that should work...

**Comment by armin-pfaeffle** [comment]
Should be fixed with version 0.3
